docs/tokens.md

```diff
@@ -0,0 +1,25 @@
+# Token Authorization
+
+## Installing
+
+On a mac:
+```
+brew tap mike-engel/jwt-cli
+brew install jwt-cli
+```
+
+## Create a token
+
+*note* the secret must be the same secret that TACO is configured with
+(in the `TACO_SECRET_KEY` variable)
+```
+jwt encode --secret superSeckeret --iss hydrus.stanford.edu --subject "lmcrae@stanford.edu"
```

Comment:
This command appears to be invalid `--subject` should be `--sub`, I get an error with --subject